.mockery.yaml

```diff
@@ -1,62 +1,34 @@
 all: True
 case: underscore
-dir: mocks
+dir: "{{.InterfaceDir}}"
+inpackage: True
 disable-version-string: True
-filename: "{{.PackageName}}_{{.InterfaceNameSnake}}.go"
+filename: "{{.InterfaceNameSnake}}_mock.go"
 mockname: "{{.InterfaceName}}Mock"
-outpkg: mocks
 recursive: True
 with-expecter: True
 
-# How to generate mocks from most preferred to least preferred:
```

Comment:
So in the end I felt it was wrong to have such a useless thinking for mocks 😅 

Hence, mocks are generated in the source package of their interface declaration and it's way more simple and natural...